com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+{
+    "app-id" : "com.abisource.AbiWord",
+    "branch" : "3.0",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "abiword",
+    "tags" : [
+       "stable"
+    ],
+    "finish-args" : [
+        "--allow=devel",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--share=network",
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "build-options" : {
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "boost",
+            "config-opts" : [
+                "--with-toolset=gcc",
+                "--with-icu"
+            ],
+            "cleanup" : [
+                "/lib/libboost_*.a"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.bz2",
+                    "sha256" : "686affff989ac2488f79a97b9479efb9f2abae035b5ed4d8226de6857933fd3b"
+                },
+                {
+                    "type" : "file",
+                    "path" : "boost-configure",
+                    "dest-filename" : "configure"
+                }
+            ]
+        },
+        "fribidi-0.19.7.json",
```

Comment:
Any reason to store it separately?

Reply:
convenience. I had it in a separate json.